### ISSUE TITLE
Are you planning to add triangles (i.e., mesh) voxelization coming from obj files?

### ISSUE THREAD

**Issue by Mitko88** [daavoo/pyntcloud#256]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Mitko88** [comment]
Are you planning to add triangles (i.e., mesh) voxelization coming from obj files?

**Comment by daavoo** [comment]
Yes we are planning to add more functionality for mesh operations, although the focus of pyntcloud are point clouds - hence the name :) - so we would probably integrate an existing Python mesh processing library .

In the meantime you can still voxelize a mesh if you first convert it to point cloud and then running the voxelization. The following example does exactly that:

https://github.com/daavoo/pyntcloud/blob/master/examples/%5Bstructures%5D%20VoxelGrid.ipynb

**Comment by Mitko88** [comment]
Thank you for the suggestion to try to covert first meshes into points. The only problem is that the point sampler is random and there are always some gaps when you voxelize the points. I'm not sure if uniform point sampling could be implemented for meshes, that would prevent those gaps I guess.
